v2/config.go

```diff
@@ -19,9 +19,8 @@ import (
 	"go.uber.org/zap/exp/zapslog"
 	"golang.org/x/exp/slog"
 
-	"github.com/libp2p/go-libp2p-kad-dht/v2/coord"
-	"github.com/libp2p/go-libp2p-kad-dht/v2/coord/routing"
-	"github.com/libp2p/go-libp2p-kad-dht/v2/kadt"
+	"github.com/libp2p/go-libp2p-kad-dht/v2/internal/coord/routing"
```

Comment:
Used for the Routing Table. I need to replicate the necessary interface here

Reply:
Defined interface in kadt package since it is part of the bridge between libp2p and go-kademlia types